### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OmkarJ13**
Improved the README to include common issues I faced while setting up teams app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,23 +11,26 @@
 
 * This will spin up a ngrok tunnel with a host name that will proxy slack's API calls to our local server.
 
-## Creating new Teams Dev app for local development ##
+## Creating a new Teams Dev app for local development ##
 1. Get or create your microsoft account - ask from abhishek/jatin (portal access), Or sign up as a developer account.
 2. Sign-in to teams desktop client, Or can also use to ms teams in browser.
-3. In the teams app store, add these two apps - "Developer Tools" and "App Studio" (this will deprecate soon).
 4. In Developer Tools, go to Tools > Bot Management > New Bot.
```
Comment (jatzz10) [inline README.md:17]: `Portal`

File: README.md [hunk 2]
```diff
@@ -11,23 +11,26 @@
...
 11. Go to App Features > Bot > Identify your bot > Select an existing bot. Select the scope as Personal. 
 12. Add these bot commands - "Link Fyle Account", and "Unlink Fyle Account".
 13. Inside the app, under Configure, go to Basic Information. Copy "App ID"
 14. Get .env creds file from anyone from Team Slack. 
 15. Inside .env file, update the TEAMS_APP_ID, TEAMS_BOT_ID, TEAMS_BOT_PASSWORD, TEAMS_SERVICE_BASE_URL (add ngrok url here) according to your local app creds.
-16. Inside this App Studio app, go to your dev app. Go to Test and distribute. Click on Add app.
+
+## Installing the Teams Dev app on your Microsoft Teams account ## 
+16. To install the app, go to Developer Portal, navigate to the 'Apps' menu and identify the app the you just created. Select 'Download app package' from the options menu.
+    - A .zip file with the name of your app should start downloading.
```
Comment (jatzz10) [inline README.md:32]: >identify the app the you just created

How will you create this the first time?

**Review by jatzz10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (OmkarJ13) [reply README.md]: oops, fixed

File: README.md (hunk 2 above)
Comment (OmkarJ13) [reply README.md:32]: This is the initial app that we use as a blueprint which specifies settings, permissions, etc.
We create this from step 9

Comment (jatzz10) [reply README.md:32]: then please mention this as well in readme

**Review by jatzz10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (OmkarJ13) [reply README.md:32]: yes you're right, fixed this

**Review by jatzz10 — APPROVED** [review]
lgtm omkar ji 🍏

**Pull request merged**